web/src/pages/Settings/Streams/StreamModal/StreamModal.jsx

```diff
@@ -7,62 +7,63 @@ import "./StreamModal.scss";
 import StreamTemplates from "../StreamTemplates.json";
 import ModalCard from "@/components/ModalCard/ModalCard";
 import TextField from '@mui/material/TextField';
+import Button from '@mui/material/Button';
+import Checkbox from '@mui/material/Checkbox';
 
 const NAME_DESC =
   "This name can be anything - it will be used to select this stream from the source selection dropdown";
 const DISABLED_DESC = "Don't show this stream in the input dropdown";
 const RESTART_DESC = "Sometimes the stream gets into a bad state and neds to be restarted. If that happened to this stream, click this to restart the stream.";
 
-const TextArg = ({ name, desc, type="text", defaultValue, onChange, required, error }) => {
+const TextArg = ({ name, desc, type="text", value, onChange, required, error }) => {
 
     return (
         <>
             <div className="stream-field">
                 <TextField
+                    className="stream-field-input"
                     type={type}
                     label={name}
                     error={error}
-                    defaultValue={defaultValue}
+                    value={value ? value : ''}
```

Comment:
https://stackoverflow.com/questions/68480218/material-ui-textfield-label-does-not-move-up-when-the-value-for-textfield-is-set